08_deploy_bmo.sh

```diff
@@ -6,6 +6,12 @@ source logging.sh
 source common.sh
 eval "$(go env)"
 
+# Set default value for provisioning interface
+PROV_IF=${PRO_IF:-ens3}
```

Comment:
Would this be clearer if we called it something like CLUSTER_PRO_IF?  Particularly since it's defaulting to the provisioning host PRO_IF, having a slightly different name for a different interface could be a bit confusing for folks that don't know the scripts well?